### DISCUSSION TITLE
Great Update!

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8843] [appreciation]
omg Monika's grad speech was so great! Also wanted to say good job on the sprites, they are perfect now.

Thanks for all the hard work! <3

**Comment by Ulvsblakk** [comment]
Wholesome content <3

**Comment by Thimeo** [comment]
The time limit to answer totally got me, it's a fun thing to do to and I would like to see it more in future topics. By the way, I tried to apologize to her for missing the answer the first time, but when I clicked on "I want to apologize" there was only "... for something", would have been good if it gave us the option to apologize for this specific reason.

**Comment by jmwall24** [comment]
you can go thru the speech again, just make sure you answer the question this time lol

**Comment by MercuryFuchsia** [comment]
Where did @Hapilone go? I'd assume that they're fine, but I still don't like it.
